src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBase.cpp

```diff
@@ -74,6 +73,27 @@ CompositionalMultiphaseBase::CompositionalMultiphaseBase( const string & name,
     setInputFlag( InputFlags::OPTIONAL ).
     setDescription( "Use mass formulation instead of molar" );
 
+  this->registerWrapper( viewKeyStruct::solutionChangeScalingFactorString(), &m_solutionChangeScalingFactor ).
+    setSizedFromParent( 0 ).
+    setInputFlag( InputFlags::OPTIONAL ).
+    setApplyDefaultValue( 0 ).
+    setDescription( "Damping factor for solution change targets" );
+  this->registerWrapper( viewKeyStruct::targetPressureChangeString(), &m_targetPressureChange ).
+    setSizedFromParent( 0 ).
+    setInputFlag( InputFlags::OPTIONAL ).
+    setApplyDefaultValue( 1.37895e7 ). // 2000 psi (taken from user guide)
```

Comment:
We don't really specify units in GEOSX   so one could easily run the code considering pressures to be in `bar`, for example, and then adjust all other properties accordingly. This, would make this value kinda meaningless. It's also trued that it's an optional parameter though.

Reply:
Yes, you are right. I can just replace the `targetPressureChange` (absolute) with a `targetRelativePressureChange`, that will solve this issue.